packages/react/src/stories/Callout.stories.tsx

```diff
@@ -0,0 +1,153 @@
+import { Story, Meta } from "@storybook/react";
+import {
+  Title,
+  Subtitle,
+  Description,
+  Primary,
+  ArgsTable,
+  Stories,
+  Subheading,
+} from "@storybook/addon-docs";
+import { Callout } from "../components/Callout";
+import { CalloutProps } from "../components/Callout/Callout.props";
+import calloutArgs from "../components/Callout/Callout.args";
+
+const themeDoc = `
+By changing the \`alert\` prop you can change base coloring of the links. By default this is set to \`info\`.
+
+| Theme   |  Description  |
+|----------|-------------|
+| \`info\` | Callout color and icon for info block. |
+| \`error\` | Callout color and icon for error block. |
+| \`success\` | Callout color and icon for success block. |
+| \`warning\` | Callout color and icon for warning block. |
+`;
+
+/**
+ * Callout Story
+ *
+ */
+export default {
+  title: "Components/Callout",
+  component: Callout,
+  argTypes: {},
+  parameters: {
+    componentSubtitle: "Component",
+    docs: {
+      page: () => (
+        <>
+          <Subtitle />
+          <Title />
+          <Description>
+            The link component creates an anchor link with a specific theme.
```

Comment:
This is still the Link description - maybe something like: The Callout component provides an inline alert or message to end users.

Reply:
Fixed the copy